src/Api/Controllers/OrganizationsController.cs

```diff
@@ -143,6 +148,26 @@ public async Task<ListResponseModel<ProfileOrganizationResponseModel>> GetUser()
             var responses = organizations.Select(o => new ProfileOrganizationResponseModel(o));
             return new ListResponseModel<ProfileOrganizationResponseModel>(responses);
         }
+        
+        [HttpGet("{identifier}/auto-enroll-status")]
+        public async Task<OrganizationAutoEnrollStatusResponseModel> GetAutoEnrollStatus(string identifier)
+        {
+            var organization = await _organizationRepository.GetByIdentifierAsync(identifier);
```

Comment:
Should probably validate the user is as least a member of the org, too

Reply:
Updated!